lib/model/internal_link.dart

```diff
@@ -159,6 +161,7 @@ Narrow? _interpretNarrowSegments(List<String> segments, PerAccountStore store) {
 
   ApiNarrowStream? streamElement;
   ApiNarrowTopic? topicElement;
+  ApiNarrowWith? withElement;
   ApiNarrowDm? dmElement;
```

Comment:
nit: put these, and the cases below, in the same order as each other and as the respective ApiNarrowElement subclasses